### PR TITLE
first draft of the 2018 survey [Do not merge]

### PR DESCRIPTION
**Pull request by halkeye**
Finally able to sit at a desk pretty regularly, so wanted to get a draft going of what's been going in my head so people can review.

### CODE REVIEW

File: correspondence/survey2018.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* How was Elysian Coffee Venue?
+    *  1 - 10 / NA
+    *  Additional feedback/notes
+
+*  .... maybe preference for value location? Can we do ordered list?
+    * Vancouver 1-10
+    * Burnaby 1-10
+    * Surrey 1-10
+    * Richmond 1-10
+    * Other
```
Comment (Qard) [inline correspondence/survey2018.md:36]: Should we include New West and Coquitlam?

**Comment by Qard** [comment]
It would also be good to get feedback on timing: weekends versus weekdays, shorter/longer/same length, etc.

File: correspondence/survey2018.md (hunk 1 above)
Comment (halkeye) [reply correspondence/survey2018.md:36]: Do we have any leads on venues there?

I know @emclaren was talking about sfu surrey at some point, and i kinda assumed richmond library might be interested after the C&C stuff.

**Comment by halkeye** [comment]
> It would also be good to get feedback on timing: weekends versus weekdays, shorter/longer/same length, etc.

Awesome, I'll get those added too

File: correspondence/survey2018.md (hunk 1 above)
Comment (Qard) [reply correspondence/survey2018.md:36]: Ah, nope. Nothing off the top of my head.

Comment (halkeye) [reply correspondence/survey2018.md:36]: Btw, not a No, just was my original thinking

**Comment by halkeye** [comment]
Also feel like we should add something along the lines of "Did you know we had a slack/github/twitter/etc"

File: correspondence/survey2018.md (hunk 1 above)
Comment (Qard) [reply correspondence/survey2018.md:36]: Yeah, I'm good either way.

**Comment by Qard** [comment]
👍 for directing people to our various communication channels. :)

File: correspondence/survey2018.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    * yes/no
+* Did you attend the basic workshop?
+    * yes/no
+* Did you attend the open source workshop?
+    * yes/no
+
+Locations
+----
+
+* How was Sauce Labs Venue?
```
Comment (keywordnew) [inline correspondence/survey2018.md:21]: nit: "as a venue?"

File: correspondence/survey2018.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    *  1 - 10 / NA
+    *  Additional feedback/notes
+* How was Clio as a Venue?
+    *  1 - 10 / NA
+    *  Additional feedback/notes
+* How was Elysian Coffee Venue?
```
Comment (keywordnew) [inline correspondence/survey2018.md:27]: nit: "as a venue?"

File: correspondence/survey2018.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    *  1 - 10 / NA
+    *  Additional feedback/notes
+
+*  .... maybe preference for value location? Can we do ordered list?
```
Comment (keywordnew) [inline correspondence/survey2018.md:31]: Love this 💯 We totes need the data.

**Review by keywordnew — COMMENTED** [review]
Looks great to me. Left a few comments with nits that can be copy pasted in if the suggested are acceptable.

**Comment by keywordnew** [comment]
@joslinchan could we get your input on this doc? Specifically looking at lines 43 through 46. Totes ok to comment on the lines.

Clarifying questions on the content, purpose, or process are also welcome!

**Comment by daffl** [comment]
We did a similar survey in Calgary in 2015 (https://docs.google.com/forms/d/e/1FAIpQLSfs4DihROp2usJHtw8FXTZVr3WgatLbvv_is8elUPJRSw24Rg/viewform) and phrased something similar to 43+ like

_I think I learn best..._

- Watching a presentation
- Walking through code
- Copy & Pasting from the internet
- Working on a problem in a small team
- Reading a tutorial or book

Side note, the results back then looked like this:

i found the meetup topics so far

**Comment by halkeye** [comment]
so outstanding issues

1) https://docs.google.com/forms/d/e/1FAIpQLSfLVYlUAcdizDUlNig74vFq9EJ8ESDuwn6PCPwBR9thadk74g/viewform

I updated it based on the feedback above, @Qard @chowdhurian do you want to sign off on it?

2) Do you want me to update the markdown here with the final version and merge it?

3) Do anyone feel strongly (or even non strongly) about pulling any questions from the survey david linked to add it to our own?

Gavin

**Review by keywordnew — APPROVED** [review]
Just putting in explicit sign off for reference.

It's labelled "Do Not Merge" and the survey is in the wild 💯

**Pull request closed without merging**